### PR TITLE
docs(addon-editor): `Editor` fix `max-height`-description (API-page)

### LINKED ISSUE #2677: 📚 - Rich Text Editor API documentation
### What is the affected URL?

https://taiga-ui.dev/components/editor/API

### Description

`[max-height]` config description and type contains copy-pasted description and type from `[exampleText]` config.

image

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by dublinsubway**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [X] Documentation content changes

## What is the current behavior?

Closes #2677

## What is the new behavior?

Uses correct hint for maxheight parameter

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/modules/components/editor/editor.template.html [hunk 1]
```diff
@@ -185,7 +185,7 @@ <h4>Text:</h4>
                 documentationPropertyType="string"
                 [(documentationPropertyValue)]="maxHeight"
             >
-                Example text shown on empty focused input
+                Example maximum height
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor/editor.template.html:188]: Thanks for your contribution!

Let's add more improvements:
```html
<ng-template
    i18n
    documentationPropertyName="style.max-height.px"
    documentationPropertyMode="input"
    documentationPropertyType="number"
    [(documentationPropertyValue)]="maxHeight"
>
    Value of CSS-property "max-height"
</ng-template>
```

**Comment by dublinsubway** [comment]
Cheers for suggesting, apologies for not clarifying what's required from me earlier! Updated the branch.

**Review by nsbarsukov — APPROVED** [review]

**Comment by nsbarsukov** [comment]
@dublinsubway 
`E2E tests / addon-editor (pull_request)`-job failed(

Possibly, it happens bacause of this line:
https://github.com/Tinkoff/taiga-ui/blob/4b971457a4111497684a9123964d7a8eaf4da91d/projects/demo-integrations/cypress/support/editor/helpers.ts#L17

**Review by waterplea — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]
🚀

**Pull request merged**